roles/sap_ha_cluster/tasks/construct_vars_hana_scaleup.yml

```diff
@@ -0,0 +1,13 @@
+---
+# Variables containing variables must be constructed with values
+# to be fed into an included role
+
+# TODO: add here any scale-up special variable constructions
+# Make sure to first respect 'ha_cluster' native variables
+
+#- name: "{{ __tname }} Construct cluster vars for SAP HANA Scale-up"
+#  ansible.builtin.set_fact:
+#
+- name: "{{ __tname }} Info"
+  ansible.builtin.debug:
+    msg: "INFO: There is currently no Scale-up specific construction, in addition to the SAP HANA common definitions."
```

Comment:
as before:
I think the comma should be removed. Or the sentence could be replaced by:
'There is currently no Scale-out specific construction on top of the SAP HANA common definitions.'

Reply:
Can be done later.